### ISSUE TITLE
NDEF label present

### ISSUE THREAD

**Issue by sethebner** [unimorph/ara#1]
There are numerous instances of the label "NDEF" in this document, which does not appear in the UniMorph schema. Should these instances be replaced with "INDF"?

**Comment by aryamccarthy** [comment]
Yeah, can you submit a PR? (Bonus points if you also fix the templates in these files, which are used for extraction! https://github.com/unimorph/wiktionary-annotations/search?q=NDEF&unscoped_q=NDEF)
